lib/chains/pollUrlChain.js

```diff
@@ -16,6 +16,7 @@ const {
 const {pollUrl} = require('../texts');
 
 const toString = ({url, response}) => pollUrl(url, response);
+const methodToString = (data) => `pollUrl(${data.url || '...'})`;
```

Comment:
For some reason it's not validated, however instead of adding `|| '...'` I would rather made a proper validation of input parameters.